src/main/resources/swagger/api-docs.yaml

```diff
@@ -721,7 +721,7 @@ definitions:
       masterDiskSize:
         type: integer
         description: |
-          Optional, the size of the disk on the master node. Minimum size is 100GB. If unspecified, default size is 500GB.
+          Optional, the size of the disk on the master node. Minimum size is 10GB. If unspecified, default size is 100GB.
```

Comment:
Nitpick: can we say "the size in gigabytes of the disk". I feel like it's better to specify the units.